pallets/gear/src/manager/mod.rs

```diff
@@ -388,4 +364,38 @@ where
             log::trace!("Signal wasn't sent due to inappropriate supply");
         }
     }
+
+    /// Removes memory pages of the program and transfers program balance to the `value_destination`.
+    fn clean_inactive_program(program_id: ProgramId, value_destination: ProgramId) {
+        ProgramStorageOf::<T>::remove_program_pages(program_id);
+
+        let program_account = &<T::AccountId as Origin>::from_origin(program_id.into_origin());
+        let balance = CurrencyOf::<T>::free_balance(program_account);
+        if !balance.is_zero() {
+            let destination = Pallet::<T>::inheritor_for(value_destination);
+            let destination = <T::AccountId as Origin>::from_origin(destination.into_origin());
+
+            CurrencyOf::<T>::transfer(
+                program_account,
+                &destination,
+                balance,
+                ExistenceRequirement::AllowDeath,
+            )
+            .unwrap_or_else(|e| unreachable!("Failed to transfer value: {:?}", e));
```

Comment:
Please prefer united style of formatting messages.
```suggestion
            .unwrap_or_else(|e| unreachable!("Failed to transfer value: {e:?}"));
```